startup_gateway/chat/mongo_service.py

```
@@ -249,44 +178,72 @@ def mark_messages_read(
         )
         
         return result.modified_count
     
+    def mark_message_delivered(
+        self,
+        message_id: str,
+        user_id: int
+    ) -> Optional[Dict[str, Any]]:
+        try:
+            msg_object_id = ObjectId(message_id)
+        except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)